code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -1336,3 +1336,782 @@
+				target.adjustHalLoss(40)
+				target.flash_pain(40)
+				target.adjustBrainLoss(20)
+				S.Destroy()
+				host.last_action = world.time + 1 MINUTES
+			else
+				to_chat(src, SPAN_WARNING("You feel a sudden pain in your neural socket as you are repelled by [target]'s countermeasures!"))
+				to_chat(target, SPAN_WARNING("[src] attempted to void your neural socket, but your Hive's countermeasures repelled them!"))
+				src.adjustHalLoss(20)
+				src.flash_pain(20)
+				host.last_action = world.time + 30 MINUTES
+				return
+		else if(S.shielded)
+			to_chat(src, SPAN_NOTICE("You extend your will like a wrathful god, destroying [target]'s neural socket and permanently severing them from the Hivenet. Let that be a lesson to the rest."))
+			to_chat(target, SPAN_DANGER("In an instant, stabbing pain in your head blankets out everything else, as your neural socket smoulders and breaks apart. The last of the Hivenet you feel is [src]'s fury, drowning out everything else - before it is gone, forever."))
+			target.adjustHalLoss(40)
+			target.flash_pain(40)
+			target.adjustBrainLoss(20)
+			S.Destroy()
+			host.last_action = world.time + 1 MINUTES
+
+/mob/living/carbon/human/proc/hivemute() //Prevents a Vaurca from speaking Hivenet, though they can still hear it
+	set name = "Mute Hivenet User"
+	set desc = "Prevent a Vaurca of your own Hive from speaking on the Hivenet, though they can still hear it. Use this ability again to unmute them. Mouv Ta may attempt to use this on Vaurca of another Hive, though this may have severe diplomatic consequences!"
+	set category = "Hivenet"
+	var/list/available_vaurca = list()
+	var/list/fullname = splittext(src.name, " ")
+	var/surname = fullname[2]
+	var/obj/item/organ/internal/vaurca/neuralsocket/host = src.internal_organs_by_name[BP_NEURAL_SOCKET]
+	if(!(all_languages[LANGUAGE_VAURCA] in src.languages) || !istype(host))
+		to_chat(src, "<span class='danger'>Your mind is dark, unable to communicate with the Hive.</span>")
+		return
+	if(!istype(src.internal_organs_by_name[BP_NEURAL_SOCKET], /obj/item/organ/internal/vaurca/neuralsocket/admin))
+		to_chat(src, "<span class='warning'>You lack the authority to do this!.</span>")
+		return
+	for(var/mob/living/carbon/human/player in player_list)
+		if(isvaurca(player) && player.internal_organs_by_name[BP_NEURAL_SOCKET])
+			var/list/player_fullname = splittext(player.name, " ")
+			var/player_surname = player_fullname[2]
+			if(player_surname == surname || src.origin.name == "Mouv Brood")
+				LAZYADD(available_vaurca, player)
+	LAZYREMOVE(available_vaurca, src) //can't ban/mute/void yourself
+	var/mob/living/carbon/human/target = input(src, "Select a Vaurca to mute.", "Hivenet Mute") as null|anything in available_vaurca
```

Comment:
There's a potential issue here in that you can use this on antag offship vaurca, even Lii'dr, which I don't think should be the case, at least for the latter

Reply:
> There's a potential issue here in that you can use this on antag offship vaurca, even Lii'dr, which I don't think should be the case, at least for the latter

This is why the shield implant was added to the uplink